docs/pages/introduction/managed-vs-bare.md

```diff
@@ -5,51 +5,39 @@ sidebar_title: Workflows
 
 import { YesIcon, NoIcon } from '~/ui/components/DocIcons';
 
-The two approaches to building applications with Expo tools are called the "managed" and "bare" workflows.
+You can use the [Expo Prebuild](/workflow/prebuild) feature to write your app entirely in JavaScript and continuously generate the native projects.
 
-- With the _managed workflow_ you only write JavaScript / TypeScript and Expo tools and services take care of everything else for you.
-- In the _bare workflow_ you can use any Expo library and service, and you are responsible for the native iOS and Android projects.
+Prebuilding is [completely optional](/workflow/prebuild#optionality) and it works great with all tools and services offered by Expo. We created the designation _bare workflow_ to refer projects that do not use `npx expo prebuild` — projects where developers make direct changes to their native projects, rather than continuously generating them on demand as with prebuild.
 
-> **If you've used React Native without any Expo tools** then you have used the "bare workflow", but the name probably doesn't sound familiar. It's easier to talk about something when it has a name, so we call this "bare" – somewhat in jest, and because of the existing term "bare metal". If you have direct access to the native code it's a _bare_ project. The ["Already used React Native?"](/workflow/already-used-react-native) page might be useful for you to quickly understand where Expo fits in.
+We used to refer to the usage of Expo Prebuild as _managed workflow_ in older versions of Expo that utilized shell apps, this term is being phased out.
```

Comment:
given this is an intro doc we shouldn't talk about what this used to be, except possibly in a parenthetical or collapsed section

Reply:
Maybe `shell apps` here is also too technical for people outside of Expo. How about "in older versions of Expo that were limited to Expo Go only"? It's not fully correct since we aren't using Expo Go in turtle v1. But I think it might make more sense for users vs shell apps.